src/AWS.Deploy.CLI/ConsoleOrchestratorLogger.cs

```diff
@@ -28,5 +33,15 @@ public void LogDebugLine(string message)
         {
             _interactiveService.WriteDebugLine(message);
         }
+
+        public void ErrorDataReceived(ProcessStartInfo processStartInfo, string data)
+        {
+            LogMessageLine(data);
```

Comment:
Shouldn't this be calling "LogErrorMessageLine"?

Reply:
https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.CLI/Utilities/CommandLineWrapper.cs#L88

we don't use `LogErrorMessageLine` in current version of `CommandLineWrapper`. Intention is to not change any behavior in refactor PR.